section2/reflection.md

```diff
@@ -1,29 +1,71 @@
 ## Section 2 Reflection
 
 1. Regarding the blog posts in Part A, how do you feel about asking questions? Do you tend to ask them too soon, or wait too long, or somewhere in between?
+I have elements of both problems. I have one friend who can help me with some of this, and I probably ask her too early. I will google the problem and try a few fixes, but if the first handful don't work, I try to ask her. Otherwise, I tend to wait too long. I have yet to really ask a public/slack question. I worry about revealing how behind I am.
 
 ### If Statements
 
-1. What is a conditional statement? Give three examples.
+1. What is a conditional statement? Give three examples.  
```

Comment:
This was asking for three examples of conditional statements, like when would you use a conditional in the real world or in code?

Reply:
But great work breaking down the parts of an if-statement :)